Cargo.lock

```diff
@@ -11,6 +11,38 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "android-tzdata"
```

Comment:
Quite a few of these additional deps are unnecessary - I've opened a PR against `opentelemetry-stdout` to prevent them:
https://github.com/open-telemetry/opentelemetry-rust/pull/1569